server/src/services/blob_storage/filesystem_impl.ts

```diff
@@ -170,4 +170,25 @@ export default class Filesystem implements BlobStorage {
         this._directory = directory;
         this._isWindows = isWindows;
     }
+
+    renameFile(f: File): Promise<Result<boolean>> {
+        try {
+            // eslint-disable-next-line security/detect-non-literal-fs-filename -- names are generated by the server
+            const rename_res = fs.rename(
+                `${f.adapter_file_path}${f.file_name}${f.short_uuid}`,
+                `${f.adapter_file_path}${f.short_uuid}${f.file_name}`,
+                (err) => {
+                    if (err) throw err; // try/catch this
```

Comment:
is this comment a TODO that never got done? If the error handling is sufficient, should we remove this comment?

Reply:
it was, I didn't put `// todo` like I usually do so I missed that one